openwisp_monitoring/monitoring/migrations/0001_squashed_0023_alert_settings_tolerance_remove_default.py

```diff
@@ -48,9 +49,12 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('monitoring', '__first__'),
-        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        swapper.dependency(*swapper.split(settings.AUTH_USER_MODEL), latest=True),
         ('contenttypes', '0002_remove_content_type_name'),
-        ('openwisp_notifications', '0003_notification_notification_type'),
+        dependency(
+            *split(settings.AUTH_USER_MODEL),
```

Comment:
This is quite wrong, it should point to the notifications module, not the user model. See how we point to other modules in other migrations in the other openwisp modules.

Reply:
Okay! I will learn and would update.

Reply:
Would it be pointing to `OPENWISP_NOTIFICATIONS_NOTIFICATION_MODEL`?

Reply:
@yashikajotwani12 sorry for my late response, feel free to ping me in the dev chat if I don't reply.

Yes I think we should do that, what do you think @pandafy?

Reply:
I would surely follow this!

Reply:
I think `swapper.dependency('openwisp_notifications', 'Notifications')` would do.

Reply:
Okay! I will learn more about it.